openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -633,7 +633,6 @@
         "profileSample": {
           "description": "Percentage of data we want to execute the profiler and tests on. Represented in the range (0, 100).",
```

Comment:
Can you update the description, there is still range (0, 100)

Reply:
Resolved